x-pack/plugins/ml/public/application/services/ml_api_service/index.ts

```diff
@@ -242,6 +255,13 @@ export const ml = {
     });
   },
 
+  forceStopDatafeed({ datafeedId }: { datafeedId: string }) {
```

Comment:
There is already a `stopDatafeed` method. You should probably just extend it with a new param.